### PR TITLE
Add quality assurance section to quality control lesson

### PR DESCRIPTION
**Pull request by ethanwhite**
There are two stages to making sure that the data entered in a dataset is correct.
The first of these  is quality assurance, which are approaches that prevent bad data
from ever getting into the dataset in the first place. This change briefly explains
the different between quality assurance and control and then shows how to implement
quality assurance in spreadsheets.

### CODE REVIEW

**Comment by ethanwhite** [comment]
Rendered version here:
https://github.com/ethanwhite/spreadsheet-ecology-lesson/blob/qaqc/04-quality-control.md

File: 04-quality-control.md [hunk 1]
```diff
@@ -1,14 +1,83 @@
...
 
-When you have a well-structured data table, you can use several simple techniques within your spreadsheet to ensure the data you’ve entered is free of errors. 
+When you have a well-structured data table, you can use several simple
+techniques within your spreadsheet to ensure the data you enter is free of
+errors. These approaches include techniques that are implemented prior to entering
+data (quality assurance) and techniques that are used after entering data to
+check for errors (quality control).
+
+# Quality Assurance
+nc
```
Comment (fmichonneau) [inline 04-quality-control.md]: remove nc?

**Comment by fmichonneau** [comment]
That looks good to me. +1

File: 04-quality-control.md (hunk 1 above)
Comment (ethanwhite) [inline 04-quality-control.md]: No idea how that got there :confused:. Thanks for catching it. Fixed.

**Comment by tracykteal** [comment]
Thanks for this additions @ethanwhite

**Pull request merged**